python/src/lib/treasury_generation_common.py

```diff
@@ -22,6 +23,17 @@ class UserObj(BaseModel):
     id: int
     email: str
 
+class OutputFileType(Enum):
+    XLSX = "xlsx"
+    CSV = "csv"
+    JSON = "json"
+
+
+def get_generated_output_file_key(
+    file_type: OutputFileType, project: str, organization: OrganizationObj
```

Comment:
any appetite for making `project` values an Enum too? I know we're grabbing from the dictionary below so perhaps not strictly necessary but could make the flow here more legible?

Reply:
💯  - I'll create a separate PR with this updated